nibabel/tests/test_wrapstruct.py

```diff
@@ -120,15 +119,17 @@ def get_bad_bb(self):
         return None
 
     def test_general_init(self):
+        if not self.header_class:
+            pytest.skip()
```

Comment:
I guess `nose` was skipping `_TestWrapStructBase`, but `pytest` isn't... Possibly we can do something with `setUp()` to ensure that the base class doesn't run but the subclasses do.

Another option would be to provide an absolute minimal `header_class` to test on.